docs/writing_planemo_galaxy_wrapper.rst

```diff
@@ -0,0 +1,222 @@
+*Galaxy wrapper generation with Planemo*
+
+
+**************************
+Prerequisites
+**************************
+Installation of 
```

Comment:
Unless this stuff is different from Planemo's documentation, we probably shouldn't repeat it here in case we go out of synch with them.